### ISSUE TITLE
[Need] Is there any way to copy the specific property in the right board

### ISSUE THREAD

**Issue by ambodu** [fkling/astexplorer#714]
![企业微信截图_17124612225366](https://github.com/fkling/astexplorer/assets/72451061/17db260e-5fc3-455d-87b3-7aaf4ff11433)

I have to write a long code if I can't copy !

So, is there any way to copy or we can make this feature:)

**Comment by Scipion** [comment]
No, I don't think it's possible to copy, but nice idea for a feature.

but, what should be the format of the text copied?
I believe that should be discussed.
